components/brave_shields/content/browser/brave_shields_util.cc

```diff
@@ -156,8 +156,7 @@ std::string ControlTypeToString(ControlType type) {
     case ControlType::DEFAULT:
       return "default";
     default:
-      NOTREACHED_IN_MIGRATION();
-      return "invalid";
+      NOTREACHED_NORETURN();
```

Comment:
We could remove the default case and add the following code after the switch statement. This way, if any new types are added in the future, a compiler warning will be generated, preventing potential browser crashes.

```
  NOTREACHED_NORETURN() << "Unexpected value for ControlType: "
                        << base::to_underlying(type);
```
```